### ISSUE TITLE
Conflicts with babelify + es2015 preset?

### ISSUE THREAD

**Issue by mattdesl** [sheetify/sheetify#49]
Getting the following when `babelify` is the first transform. Does not occur when `babelify` is defined after `sheetify`.

```
path.js:8
    throw new TypeError('Path must be a string. Received ' +
    ^

TypeError: Path must be a string. Received undefined
    at assertPath (path.js:8:11)
    at Object.posix.join (path.js:479:5)
    at walk (/projects/mattdesl.github.com/node_modules/sheetify/transform.js:144:14)
    at walk (/projects/mattdesl.github.com/node_modules/sheetify/node_modules/falafel/index.js:49:9)
    at /projects/mattdesl.github.com/node_modules/sheetify/node_modules/falafel/index.js:46:17
    at forEach (/projects/mattdesl.github.com/node_modules/sheetify/node_modules/falafel/node_modules/foreach/index.js:12:16)
    at walk (/projects/mattdesl.github.com/node_modules/sheetify/node_modules/falafel/index.js:34:9)
    at /projects/mattdesl.github.com/node_modules/sheetify/node_modules/falafel/index.js:41:25
    at forEach (/projects/mattdesl.github.com/node_modules/sheetify/node_modules/falafel/node_modules/foreach/index.js:12:16)
    at /projects/mattdesl.github.com/node_modules/sheetify/node_modules/falafel/index.js:39:17
```

**Comment by yoshuawuyts** [comment]
ah yeah, so `sheetify` specifically looks for template strings in the code. If it's a regular string it'll try and either resolve it from `node_modules` or try and read it locally. adding `babel` beforehand breaks this behavior. I think there's 2 things that should be done:
- [ ] if a path is not resolved (e.g. `fs.stat` to verify) return a meaningful error message
- [ ] add a note about this in the documentation so people down the road can read up on this beforehand

Does this make sense?

**Comment by mattdesl** [comment]
I think the problem is that babelify + es2015 preset will transform the sheetify-tagged template string into a regular string... If that's the case, it might be tricky to solve. :confused: Putting babelify after the sheetify transform isn't always possible, e.g. if you have JSX or something else that would break sheetify's AST parser.

**Comment by yoshuawuyts** [comment]
I think this should be ok, https://github.com/substack/node-browserify/issues/1427 implies that `.jsx` files can still be found and parsed correctly by `browserify` out of the box. This means that if `sheetify` is called before `babel` no problems should occur.

**Comment by yoshuawuyts** [comment]
Should be resolved in #52
